viewer/packages/360-images/src/icons/IconCollection.ts

```diff
@@ -8,6 +8,11 @@ import { Image360Icon } from './Image360Icon';
 import { InstancedIconSprite } from './InstancedIconSprite';
 import { IconOctree } from './IconOctree';
 
+export enum IconCullingScheme {
+  Clustered,
+  Proximity
+}
```

Comment:
Concider changing this to `export type IconCullingScheme = 'clustered' | 'proximity';`

In any other language, using an enum here is correct. But typescript's enums are notoriously bad (here is a talk on it if you are interested in why: https://www.youtube.com/watch?v=0fTdCSH_QEU).

Also you would be able to leverage duck-typing here such that a user can do:
```ts
setCullingScheme('clustered');
```

or
```ts
const clusterScheme = 'clustered';
setCullingScheme(clusterScheme);
```
such that they don't have to import the type unless they want to.